tests/test_ssd1306.py

```diff
@@ -76,6 +76,28 @@ def test_init_96x16():
     serial.data.assert_called_once_with([0] * (96 * 16 // 8))
 
 
+def test_init_64x48():
```

Comment:
there should be a single test that tests all of the supported dimensions (someday).